.travis.yml

```diff
@@ -44,4 +44,4 @@ notifications:
   email: true
 
 after_script:
-  - npm run server:cov:publish
+  - npm run server:cov
```

Comment:
Ah, I see you combined those tasks, I actually used that server:cov to look into coverages locally. Well I can work around it if nobody else needs it